src/planner/kinodynamic/KinodynamicTimer.cpp

```diff
@@ -0,0 +1,273 @@
+#include "aikido/planner/kinodynamic/KinodynamicTimer.hpp"
+#include <dart/dart.hpp>
+#include <aikido/common/Spline.hpp>
+#include <aikido/common/StepSequence.hpp>
+
+#include "Path.h"
```

Comment:
@jslee02 Are relative paths to the external files okay here or should this be something else?

Reply:
I would also like to know the standard. This one is following the style in `HauserParabolicTimer`.

Reply:
It seems this is the current convention unless we change the `CMakeLists.txt` files in the external directories because the include directories of the targets are exactly the directory where the files are located (e.g., `<aikido_dir>/src/external/hauserparabolicsmoother/`).

One potential problem with this approach is that the file name can be ambiguous when there are two files with the same name in the different external libraries. I have a proposal to resolve this problem:

```
src/
  external/
    lib1/
      CMakeLists.txt  # include ./include
      include/
        lib1/
      src/
    lib2/
      CMakeLists.txt  # include ./include
      include/
        lib2/
      src/
```
then we can include the files without conflicts over external libraries as
```cpp
#include "lib1/same_name.h"
#include "lib2/same_name.h"
```

With this proposal, the path to the external headers in the external sources should be changed as the same way above. There is a way to avoid this modification, but I personally prefer this because the leading project name in the header inclusion would also allow the externals to avoid potential name conflicts when they also have dependency.

TL;DR: This is the convention. I have a proposal, which shouldn't block this PR anyway.